frontend/tests/e2e/specs/student/tournament.js

```diff
@@ -0,0 +1,97 @@
+describe('Student walkthrough', () => {
+
+  beforeEach(() => {
+    cy.demoStudentLogin();
+    cy.addQuestionTopic();
+  });
+
+  afterEach(() => {
+    cy.exec(
+      'PGPASSWORD=' +
+      Cypress.env('PASS') +
+      ' psql -d ' +
+      Cypress.env('DBNAME') +
+      ' -U ' +
+      Cypress.env('USER') +
+      ' -h localhost -c "DELETE FROM topics_questions WHERE questions_id = 1389;' +
+      ' DELETE FROM tournaments_participants;' +
+      ' DELETE FROM tournaments;' +
+      ' ALTER SEQUENCE tournaments_id_seq RESTART WITH 1;' +
+      ' UPDATE tournaments SET id=nextval(\'tournaments_id_seq\');" '
+    );
+    cy.contains('Demo Course').click();
+    cy.logout();
+  });
+
+  it('login sees all tournaments', () => {
+    cy.seeTournamentsLists('All');
```

Comment:
You are not actually validating/testing anything. What you are doing is seeing if no error happen, but you should use the "should" validation to make sure that you are in the correct page or has the correct number of tournaments